rest_framework-stubs/renderers.pyi

```diff
@@ -13,8 +13,8 @@ from rest_framework.views import APIView
 def zero_as_none(value: Any) -> Any | None: ...
 
 class BaseRenderer:
-    media_type: str
-    format: str
+    media_type: str | None
```

Comment:
If you are to make this change - you should also change all its subclasses: https://github.com/encode/django-rest-framework/blob/dbac145638758413b966c3418fa5f3f651e3e02a/rest_framework/renderers.py#L42-L48

They have to override `None` to some `str` value.

Reply:
Thanks, I must have missed something. 
Thanks for checking it out.